vcluster/deploy/environment/eks.mdx

```diff
@@ -0,0 +1,210 @@
+ebs-csi-node-jg8n8                   3/3     Running   0          2m14s
+```
+
+### Configure storage class
+
+`vCluster` requires a default StorageClass for its persistent volumes. EKS provides the `gp2` StorageClass by default, but `gp3` is recommended for better performance and cost efficiency. Create a new StorageClass:
+
+```bash title="gp3 StorageClass configuration"
+cat <<EOF | kubectl apply -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: gp3
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: ebs.csi.aws.com
+parameters:
+  type: gp3
+  encrypted: "true"
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+EOF
+```
+
+Remove the default status from the `gp2` StorageClass:
+
+```bash title="Remove default status from gp2 StorageClass"
+kubectl patch storageclass gp2 -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}'
+```
+
+### Configure workload identity
+
+Now create a service account in your `vCluster` that uses Workload Identity:
+
+```yaml title="vCluster workload service account"
+cat << 'EOF' | kubectl apply -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: vcluster-workload-sa
+  namespace: default
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::$(aws sts get-caller-identity --query Account --output text):role/vcluster-pod-role
+EOF
+```
+
+### Verify the Installation
+
+Check if `vCluster` pods are running:
+
+```bash title="Check vCluster pods"
+kubectl get pods -n team-x
+```
+
+You should see output similar to:
+```
+NAME                                                   READY   STATUS    RESTARTS   AGE
+coredns-666d64755b-k5njg-x-kube-system-x-my-vcluster   1/1     Running   0          3m11s
+my-vcluster-0                                          1/1     Running   0          6m33s
+```
+
+This configuration ensures that:
+- Service accounts are properly synced between virtual and host clusters
+- Persistent volume claims are handled correctly
+- The `gp3` storage class created earlier is used
+
+
+## Create virtual cluster
+
+Now that all the prerequisites are configured, create a virtual cluster
+using the CLI.
+
+```yaml title="vCluster values configuration"
+cat << 'EOF' | vcluster create my-vcluster --namespace team-x --values -
+sync:
+  toHost:
+    serviceAccounts:
+      enabled: true
+EOF
+```
+
+<KubeconfigUpdate />
+
+## Next steps
+
+Now that you have `vCluster` running on EKS, consider exploring:
+
+### Platform UI
+
+Setting up the [platform UI](/platform/install/quick-start-guide) for `vCluster`
+
+### Pod identity
+
+<ProAdmonition />
+
+For `vCluster` Pro users, you can enable [Pod
+Identity](/vcluster/integrations/pod-identity/eks-pod-identity)) to allow pods to assume IAM roles. This feature is useful for accessing AWS services securely from your pods.
+
+## Cleanup
+
+Remember to clean up resources when you're done experimenting:
```

Comment:
I wouldn't make the assumption that they are experimenting or that this guide is for experimenting.

Reply:
The guide assumes that the user will create a fresh EKS cluster using `eksctl`, that's why _experimenting_. Otherwise, the `eksctl` configuration file would have to be split into various pieces and take into account existing clusters` config.

This also means that we expect the user to remove the cluster after the deployment is done.

Maybe we could remove the _experimenting_ part but still keep cluster creation and change it later on if needed?

Reply:
Yes, I don't like the word "experimenting" which implies this is not how you would set it up for production use cases. If we remove that, I'll be fine.